### DISCUSSION TITLE
Student account has been verified (weeks ago) but can't access any of the offers still?

### DISCUSSION THREAD

**Discussion by EthanJayC** [community/community#86581] [GitHub Education]
### Hi everyone,

So my profile is verified and good to go, I'm trying to get the Mobile App Development pack for the 6 month frontend masters offer, but when I navigate to the page (https://education.github.com/experiences/mobile_app_development) it just gives me the information, there's no way to actually access the offer. 
Yet I **am** verified? Any ideas what to do? Making an account on their website and linking it to GitHub doesn't do anything either..

Thanks
![image](https://github.com/community/community/assets/61258513/4ec30ce1-7908-419f-b16c-a847fd0195b8)

**Comment by PrashantRatanparikhe** [comment]
Hello @EthanJayC 
Please check this [link](https://github.com/orgs/community/discussions/66772#discussioncomment-7106627).
https://github.com/orgs/community/discussions/66772#discussioncomment-7106627

Thanks
